### ISSUE TITLE
AI Doesn't Launch Atom Bomb / Nuke If You Don't Have a ConYard

### ISSUE THREAD

**Issue by StevenL-Up** [OpenRA/OpenRA#19449]
## Issue Summary

If you un-deploy your ConYard into an MCV in time for the end of the countdown AI's Nuke/Atom Bomb, it will *never* launch, even if you re-deploy into a  ConYard again, and even if you have other structures or units.

Similar to https://github.com/OpenRA/OpenRA/issues/13251

Version `release-20210321`

## Reproduction

Wait for AI's Atom Bomb to reach ~10 seconds left  for AI (exact amount doesn't seem to matter)
Un-deploy ConYard into MCV
Wait for AI's Atom Bomb count-down to finish, plus an additional few seconds to be  sure
Re-deploy into ConYard

AI will never launch Nuke

## Expected behavior

EXPECTED BEHAVIOR

AI launches Nuke at end of countdown at a valid target regardless of ConYard status or existence

(I have played this map several times, and the Nuke will launch as expected if I don't un-deploy the ConYard.)

## Screenshots / Screen recordings / Replays

https://user-images.githubusercontent.com/66263703/121961279-3caf1500-cd35-11eb-9907-6265fb5482d1.mp4

**Comment by chrisharris333** [comment]
I just played through this, attempted to reproduce this several times, and was not able to.  What I observed is that the AI does not always launch the nuke right as the countdown gets to zero.  In one case, I heard the 'A-Bomb launch detected' message seconds after the countdown timer reaches zero, about a minute after in one run, and about 5 minutes after in a different run.

Are we certain that the intent is to launch the missile the instant the timer reaches zero?
